app/channel.go

```diff
@@ -186,7 +186,40 @@ func (a *App) JoinDefaultChannels(c request.CTX, teamID string, user *model.User
 		a.invalidateCacheForChannelMembers(channel.Id)
 
 		message := model.NewWebSocketEvent(model.WebsocketEventUserAdded, "", channel.Id, "", nil, "")
+		user.Sanitize(map[string]bool{})
 		message.Add("user_id", user.Id)
+		message.Add("user_create_at", user.CreateAt)
```

Comment:
Instead of passing every single property, I would just pass a "user" property with the whole user.